samples/java-webmvc/generated-docs/index.html

```diff
@@ -2337,30 +2337,9 @@ <h4 id="_query_parameters_7"><a class="anchor" href="#_query_parameters_7"></a><
 </div>
 <div class="sect3">
 <h4 id="_request_fields_9"><a class="anchor" href="#_request_fields_9"></a><a class="link" href="#_request_fields_9">2.9.4. Request fields</a></h4>
-<table class="tableblock frame-all grid-all stretch">
-<colgroup>
-<col style="width: 25%;">
-<col style="width: 25%;">
-<col style="width: 25%;">
-<col style="width: 25%;">
-</colgroup>
-<thead>
-<tr>
-<th class="tableblock halign-left valign-top">Path</th>
-<th class="tableblock halign-left valign-top">Type</th>
-<th class="tableblock halign-left valign-top">Optional</th>
-<th class="tableblock halign-left valign-top">Description</th>
-</tr>
-</thead>
-<tbody>
-<tr>
-<td class="tableblock halign-left valign-top"><p class="tableblock">command</p></td>
-<td class="tableblock halign-left valign-top"><p class="tableblock">String</p></td>
-<td class="tableblock halign-left valign-top"><p class="tableblock">false</p></td>
-<td class="tableblock halign-left valign-top"><p class="tableblock">Command to execute.</p></td>
-</tr>
-</tbody>
-</table>
+<div class="paragraph">
+<p>No request body.</p>
```

Comment:
Something changed here.

Reply:
Fixed by adjusting the `MockMvcBase` to the new ModelAttribute support.